backend/shared/shared/yrtti/yrtti_client.py

```diff
@@ -7,6 +7,13 @@
 
 
 class YRTTIClient:
+
+    # The YRTTI API only returns data for associations, so the response does not contain
+    # a field for company form.
+    # Use the YTJ "yritysmuoto" code for associations when creating the Company objects
+    ASSOCIATION_FORM_CODE = 29
```

Comment:
You have now two `ASSOCIATION_FORM_CODE = 29` definitions, one in YRTTIClient and one in factories. Would it be prudent to combine them into one or do you see some reason why not to do that?